### PR TITLE
Staging namespace update

### PR DESCRIPTION
**Pull request by pkpaulk**
As above

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-stg/resources/ecr.tf [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "cica_ecr_credentials" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=3.2"
```
Comment (mtrblmoj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-stg/resources/ecr.tf]: Can you please upgrade to the latest version of the module : 3.4 
https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/releases

Comment (pkpaulk) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-stg/resources/ecr.tf]: Updated config reference to 3.4 now

**Comment by mtrblmoj** [comment]
I see **claim-criminal-injuries-compensation-stg/** 
and **claim-criminal-injuries-compensation-staging/**   

Could you please confirm which one is correct and fix it ?

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-stg/resources/variables.tf [hunk 2]
```diff
@@ -0,0 +1,37 @@
+variable "namespace" {
```
Comment (mtrblmoj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-stg/resources/variables.tf]: this seems to be a duplicate of : 
namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-staging/resources/variables.tf

**Comment by pkpaulk** [comment]
Hi,

and claim-criminal-injuries-compensation-staging/

is the correct one.

Where do you see stg now and to resolve do you want me to remove it locally?

Kind regards,

Paul

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-stg/resources/variables.tf (hunk 2 above)
Comment (pkpaulk) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-stg/resources/variables.tf]: stg has been removed now

**Pull request merged**